chain/network/src/peer/peer_actor.rs

```diff
@@ -1020,15 +1020,14 @@ impl StreamHandler<Result<Vec<u8>, ReasonForBan>> for PeerActor {
                     // Broadcast any new data we have found, even in presence of an error.
                     // This will prevent a malicious peer from forcing us to re-verify valid
                     // datasets. See accounts_data::Cache documentation for details.
-                    // TODO(gprusak): this should be rate limited - diffs should be aggregated,
-                    // unless there was no recent broadcast of this type.
                     if new_data.len() > 0 {
-                        pms.broadcast_message(PeerMessage::SyncAccountsData(SyncAccountsData {
-                            incremental: true,
-                            requesting_full_sync: false,
-                            accounts_data: new_data,
-                        }))
-                        .await;
+                        let handles: Vec<_> = pms
+                            .connected_peers
+                            .read()
+                            .values()
+                            .map(|p| p.send_accounts_data(new_data.clone()))
+                            .collect();
+                        futures::future::join_all(handles).await;
```

Comment:
not-strong-opinion drive-by opinion: we shouldn't use the `futures` crate, we should use only `futures_util`.

Reply:
done